### ISSUE TITLE
I trained the wrong model

### ISSUE THREAD

**Issue by dj-kefir-siorbacz** [pauzii/PhaseBetweener#10]
Hi @pauzii, another issue here. 

I trained the model as per the instruction in the `README.md` (+custom Barracuda build, that I mentioned in an other issue), but I can't use it. 

The reason seems to be that it's not the same model as the `LaFAN1_150_DeepPhases` or `LaFAN1_150_LocalPhases` (`LaFAN1_150_NoPhases` doesn;t work at all so I ignore it), but rather it's more similar (in input and output sizes) to the `LaFan_150_GNN_DeepPhases_Styles` model (which produces shitty results).

My trained model is called `150` (the number of epochs, right?)
![image](https://github.com/pauzii/PhaseBetweener/assets/135321152/f3ddffc4-95a1-485b-8df0-e2c8dffd4e57).

**Comment by dj-kefir-siorbacz** [comment]
See the summary of each model here:

`150`
![image](https://github.com/pauzii/PhaseBetweener/assets/135321152/4e205c01-c11f-4725-b5a9-c151d34e2212)

`LaFAN1_150_DeepPhases`
![image](https://github.com/pauzii/PhaseBetweener/assets/135321152/da91cd0b-6537-486c-a281-b9a2a4168630)

`LaFAN1_150_LocalPhases`
![image](https://github.com/pauzii/PhaseBetweener/assets/135321152/218c2829-cf95-4cd4-84b0-34062f339ea5)


`LaFan_150_GNN_DeepPhases_Styles`
![image](https://github.com/pauzii/PhaseBetweener/assets/135321152/4ddc10c1-b6bb-434f-a20e-169235e3a075)

**Comment by dj-kefir-siorbacz** [comment]
Shouldn't we be running:
https://github.com/pauzii/PhaseBetweener/blob/main/DeepLearningONNX/Models/AEGNN/Network.py

??

**Comment by paulstarke** [comment]
Hi @dj-kefir-siorbacz !
It seems like you trained your model with style labels in the input and hence the input dimension is different from the uploaded models. 
I just updated `MotionExporter.cs` and `AuthoringInBetweeningController.cs` in the repo. Please download them again, export, and retrain. I forgot to comment out one line of code in the exporter. Sorry for the inconvenience!

 You can now choose in the exporter's UI if you want to use stylization. 
 You can test `LaFan_150_GNN_DeepPhases_Styles` in the authoring demo scene: there is a "style control" gameobject with an example path. Make sure to check "ModelIncludesStyleLabels" in the controllers inspector, so that the inputs are correctly fed into the model.
 
 If you set up everything as indented it should work now!

**Comment by dj-kefir-siorbacz** [comment]
Hi @pauzii !
Interesting. I see I can now use the `LaFan_150_GNN_DeepPhases_Styles` with `Model Includes Style Labels` checked. 
I also see that I can't use `LaFAN1_150_DeepPhases` with this option checked, due to input shape mismatch - makes sense.

I'll try to retrain and will let you know if the output is the same as in `LaFan_150_GNN_DeepPhases_Styles`.

**Comment by dj-kefir-siorbacz** [comment]
Ok, training in progress. I see that even a model trained on 1 epoch does fairly good when imported to Unity :D

However, there is one thing I don'tm understand: 

my model:
![image](https://github.com/pauzii/PhaseBetweener/assets/135321152/bf9a0c6f-a885-4077-9298-7c11edf5d267)

built-in model:
![image](https://github.com/pauzii/PhaseBetweener/assets/135321152/b61819fd-3416-4f75-966a-05a518b106f5)

Why is there such big difference in number of weights? And why layers differ?

**Comment by paulstarke** [comment]
Looking good! I think you can ignore the differences as long as the motion doesn't look broken and no errors are popping up in the console. The built-in models are actually an older version than `In-BetweeningNetwork.py`.

Again, if it doesn't work, perhaps using the same pytorch version (1.9) will help.
